test/services/messengerSync/rabbitmqMock.js

```diff
@@ -6,12 +6,16 @@ const reset = () => {
 
 const assertQueue = async () => {};
 
+const prefetch = async () => {};
+
+const consume = async () => {};
+
 const sendToQueue = (queue, messageBuffer, options) => {
 	if (!queues[queue]) queues[queue] = [];
 	queues[queue].push(messageBuffer);
 };
 
-const createChannel = async () => ({ sendToQueue, assertQueue });
+const createChannel = async () => ({ sendToQueue, assertQueue, prefetch, consume });
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected a line break before this closing brace.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5362179)